internal/cli/users.go

```diff
@@ -0,0 +1,127 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5/management"
+)
+
+var (
+	userID = Flag{
+		Name:       "User ID",
+		Help:       "Id of the user.",
+		LongForm:   "user-id",
+		ShortForm:  "u",
+		IsRequired: true,
```

Comment:
```suggestion
```

Given it's an argument, don't think we need all these other fields. Correct @Widcket ?

Reply:
Yes, and it should probably be using `Argument`, as it has the appropriate logic